src/invidious/jsonify/api_v1/video_json.cr

```diff
@@ -245,6 +245,12 @@ module Invidious::JSONify::APIv1
                 json.field "lengthSeconds", rv["length_seconds"]?.try &.to_i
                 json.field "viewCountText", rv["short_view_count"]?
                 json.field "viewCount", rv["view_count"]?.try &.empty? ? nil : rv["view_count"].to_i64
+                json.field "published", rv["published"]?
+                if !rv["published"].nil?
+                  json.field "publishedText", translate(locale, "`x` ago", recode_date(Time.unix(rv["published"].to_i), locale))
+                else
+                  json.field "publishedText", translate(locale, "`x` ago", "NaN")
+                end
```

Comment:
I'd rather not show anything if that information is unknown. It makes for a better user experience than `Il y a NaN` ("NaN ago", translated in french, for instance)